src/server/lib/okta/validateEmail.ts

```diff
@@ -10,22 +10,33 @@ import { trackMetric } from '@/server/lib/trackMetric';
  *
  * @param {string} id accepts the Okta user ID, email address (login) or login shortname (as long as it is unambiguous)
  * @param {string} ip The IP address of the user
+ * @param {boolean} unsetFlags If true, unset the emailValidated and passwordSetSecurely flags by setting them to false
  * @returns {Promise<UserResponse>} Promise that resolves to the user object
  */
-export const validateEmailAndPasswordSetSecurely = async (
-	id: string,
-	ip?: string,
-): Promise<UserResponse> => {
+export const validateEmailAndPasswordSetSecurely = async ({
+	id,
+	ip,
+	unsetFlags = false,
```

Comment:
```suggestion
	setFlags = true,
```

Perhaps just default to true and avoid the [negation on line 30](https://github.com/guardian/gateway/pull/2902/files#diff-8e0a6ee150799c6cc4fd0826332e5b6b5500e0eaef2899cad2c42dccccbc7bc8R30), I find it a little bit confusing 😅

Reply:
Yeah that's a good point! I'll see what I can conjure up.

Reply:
Updated it to `flagStatus` instead, which makes more sense than "set"/"unset"!